sigs.yml

```diff
@@ -659,4 +659,24 @@
         github: reyang
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
-        github: jpkrohling
\ No newline at end of file
+        github: jpkrohling
+    - name: Contributor Experience
+    meeting: TBA
+    notes:
+      type: gDoc
+      value: 1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0
+    chat:
+      - type: slack
+        name: '#otel-sig-contribex'
```

Comment:
Should this be #otel-contributor-experience instead?

Reply:
@mx-psi fixed, thanks